### PR TITLE
Doc/resistant color

### PR DESCRIPTION
**Pull request by artamonovkirill**
Based on feedback from https://exercism.io/mentor/solutions/b5ff011ba5164d989aefa1b5b7dbedba?iteration_idx=1, I've extended exercise explanation (partial copy from resistant-color-duo)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,4 @@
 node_modules
+
+# Intellij IDEA
+.idea
```
Comment (SaschaMann) [inline .gitignore:4]: This is user-specific and should go into `.git/info/exclude` (local version of .gitignore) instead

Comment (artamonovkirill) [reply .gitignore]: ✅ done

**Review by Stargator — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]
This is nice! Once we've opened up the prob-specs repo again (very close to doing that) we can merge this.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Thanks @artamonovkirill!

**Pull request merged**